src/index.js

```diff
@@ -129,11 +129,40 @@ for (let i=ELEMENTS.length; i--; ) {
 }
 
 
+function isStatelessComponent(c) {
```

Comment:
Thinking we should make this match [preact's existing implementation](https://github.com/developit/preact/blob/master/src/vdom/functional-component.js#L12-L15), which allows Components to not extend from Component:

```js
function isStatelessComponent(c) {
	return typeof c==='function' && !(c.prototype && c.prototype.render);
}
```